spec.bs

```diff
@@ -4329,6 +4341,11 @@ representing metrics aggregated over a particular participant (e.g. bidder or se
   :: An [=averager=].
   : <dfn>script timeouts occurred</dfn>
   :: A {{long}}, initially 0.
+  : <dfn>cumulative timeouts occurred</dfn>
+  :: A {{long}}, initially 0. Number of interest groups affected by the cumulative timeout
+    happening.
+  : <dfn>cumulative buyer time</dfn>
+  :: A [=duration=] in milliseconds, initially 0.

```

Comment:
wondering if it'll be good to duplicate what these metrics mean (as what's in signal base value), or not. Probably be good to have a very short description.

Reply:
Maybe I should link all of these to their metrics?

Reply:
yeah that'll be good if it's not too much of work.